src/cli/start.cpp

```diff
@@ -22,7 +22,8 @@ void start_help() {
     fmt::println("start help");
 }
 
-void start_args::add_cli(CLI::App& cli, global_settings& settings) {
+void start_args::add_cli(CLI::App& cli,
+                         [[maybe_unused]] global_settings& settings) {
```

Comment:
Do you have a preference between `[[maybe_unused]]` and just removing the parameter name?

Reply:
`[[maybe unused]]`, because it clearly communicates that the writer is aware of the "issue".